docker/deployed/docker-compose.observability.yml

```diff
@@ -7,6 +7,7 @@ services:
       - ${PWD}/${ENVIRONMENT}/prometheus:/prometheus/data
     command:
       - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--web.enable-admin-api'
```

Comment:
we need this to be able to delete the metrics from prometheus. see: https://prometheus.io/docs/prometheus/latest/querying/api/#delete-series